drivers/SmartThings/zigbee-switch/src/test/test_bad_device_kind.lua

```diff
@@ -0,0 +1,29 @@
+-- Copyright 2025 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local test = require "integration_test"
+local t_utils = require "integration_test.utils"
+
+local mock_device = test.mock_device.build_test_zwave_device({
+  profile = t_utils.get_profile_definition("on-off-level.yml"),
+})
+
+local function test_init()
+  test.mock_device.add_test_device(mock_device)
+end
+
+test.set_test_init_function(test_init)
+test.register_test("mismatched_prot_ignored", function() end, nil)
```

Comment:
I don't understand this test.

Reply:
I pushed up some additional comments on the test. It's just making sure the driver doesn't crash when you add a ZWave device to it.